react-ui/src/components/Reviewer/GroupNames.js

```diff
@@ -22,21 +22,25 @@ class GroupNames extends Component {
     this.setState({ names });
   }
 
+  handleUpdateReviewId(reviewId) {
+    this.props.updateReviewId(reviewId);
+  }
+
```

Comment:
Here,  we are just giving the GroupName component the power to inherit a function from its parent that will update the state for reviewId